DESCRIPTION

```diff
@@ -21,21 +21,23 @@ License: GPL-3
 URL: https://github.com/inbo/inbodb, https://inbo.github.io/inbodb/
 BugReports: https://github.com/inbo/inbodb/issues
 Imports:
-    DBI,
     assertthat,
-    dbplyr,
+    DBI,
     dplyr,
     glue,
-    methods,
     odbc,
     rlang
 Suggests: 
+    dbplyr,
     knitr,
+    methods,
```

Comment:
I'm not sure our implementations of the methods dbDisconnect and dbFetch are still functioning correctly if users haven't installed this?  But I can have a look at this while solving issue #52.  Maybe move it back to imports for now, if possible?

Reply:
The vignettes work, so I don't think there is a problem with `dbDisconnect`. Putting them in imports will cause a checklist error or warning that these packages are imported but not used.